src/os/bluestore/BitAllocator.cc

```diff
@@ -1311,6 +1311,7 @@ BitAllocator::shutdown()
 {
   lock_excl();
   serial_lock();
+  serial_unlock();
```

Comment:
what is this method doing? I am not able to get anything except lock() and unlock(). If we want to make sure no one is holding the lock then we should make sure, with a bool or counter.